client/my-sites/marketing/tools/google-adwords.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import page from 'page';
+import React, { Fragment, FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import Button from 'components/button';
+import { getCurrentUserCountryCode } from 'state/current-user/selectors';
+import { getSelectedSite } from 'state/ui/selectors';
+import GoogleVoucherDetails from 'my-sites/checkout/checkout-thank-you/google-voucher';
+import { isPremium, isBusiness, isEcommerce, isEnterprise } from 'lib/products-values';
+import isSiteAtomic from 'state/selectors/is-site-automated-transfer';
+import MarketingToolsFeature from './feature';
+import { PLAN_PREMIUM } from 'lib/plans/constants';
+import QuerySiteVouchers from 'components/data/query-site-vouchers';
+import { recordTracksEvent as recordTracksEventAction } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface ConnectedProps {
+	isAtomic: boolean;
+	isJetpack: boolean;
+	isPremiumOrHigher: boolean;
+	recordTracksEvent: () => void;
+	showCard: boolean;
+	siteId: number | null;
+	siteSlug: string | null;
+}
+
+export const MarketingToolsGoogleAdwordsFeature: FunctionComponent< ConnectedProps > = ( {
+	isAtomic,
+	isJetpack,
+	isPremiumOrHigher,
+	recordTracksEvent,
+	showCard,
+	siteId,
+	siteSlug,
+} ) => {
+	const translate = useTranslate();
+	if ( ! showCard || ( isJetpack && ! isAtomic ) || ! siteId || ! siteSlug ) {
+		return null;
+	}
+
+	const handleUpgradeClick = () => {
+		recordTracksEvent( 'calypso_marketing_tools_adwords_plan_upgrade_button_click' );
+		page( addQueryArgs( { plan: PLAN_PREMIUM }, `/plans/${ siteSlug }` ) );
+	};
+
+	const renderButton = () => {
+		if ( isPremiumOrHigher ) {
+			return <GoogleVoucherDetails />;
+		}
+		return <Button onClick={ handleUpgradeClick }>{ translate( 'Upgrade To Premium' ) }</Button>;
+	};
+
+	return (
+		<Fragment>
+			<QuerySiteVouchers siteId={ siteId } />
+			<MarketingToolsFeature
+				title={ translate( 'Advertise with your $100 Google Adwords credit' ) }
+				description={ translate(
+					"Advertise your site where most people are searching: Google. You've got a $100 credit with Google Adwords to drive traffic to your most important pages."
+				) }
+				imagePath="/calypso/images/illustrations/google-adwords-stats.svg"
+			>
+				{ renderButton() }
+			</MarketingToolsFeature>
+		</Fragment>
+	);
+};
+
+export default connect(
+	state => {
+		const userInUsa = getCurrentUserCountryCode( state ) === 'US';
+		const userInCa = getCurrentUserCountryCode( state ) === 'CA';
```

Comment:
Why do we offer this voucher to Canadian users whereas we only mention US users in the card?

![screenshot](https://user-images.githubusercontent.com/594356/57243987-b2653480-7037-11e9-850c-3ffd86dff7da.png)

Reply:
@huguesvincent Do you know the details about this offer?

Reply:
I reached back to Google to confirm but that was stating at the beginning of our back and forth.

Reply:
@stephanethomas So I think that fine print should say "US and Canada" then, but will wait for Google's confirmation from @huguesvincent.

Reply:
Confirmed by @huguesvincent on Slack.